app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find_by(id: params[:id])
+    if @quote.nil?
+      render :error
```

Comment:
this is probably more functionality than you need for now, to be honest. Just throwing the exception or dealing with some kind of flash notice later would have been fine for a commit #1